mediumProject/requirements.txt

```diff
@@ -0,0 +1,14 @@
+asgiref==3.5.2
+certifi==2022.9.24
+charset-normalizer==2.1.1
+Django==4.1.2
+djangorestframework==3.14.0
+drf-registration==0.1.3
+idna==3.4
+psycopg2-binary==2.9.5
+pytz==2022.5
+requests==2.28.1
+six==1.16.0
+sqlparse==0.4.3
+tzdata==2022.5
+urllib3==1.26.12
```

Comment:
requirements.txt의 내용물을 모두 설치해도

SystemCheckError: System check identified some issues:

ERRORS:
medium.Post.image: (fields.E210) Cannot use ImageField because Pillow is not installed.
        HINT: Get Pillow at https://pypi.org/project/Pillow/ or run command "python -m pip install Pillow".

다음과 같은 오류가 발생합니다. requirements.txt의 업데이트가 필요해 보입니다.

Reply:
pillow package을 빠뜨린 것 같네요. requirements.txt을 작성할 때 유의하겠습니다.